Sources/Sentry/SentryBacktrace.cpp

```diff
@@ -107,9 +107,6 @@ namespace profiling {
     {
         const auto pair = ThreadHandle::allExcludingCurrent();
         for (const auto &thread : pair.first) {
-            if (thread->isIdle()) {
```

Comment:
`m`: Can we somehow add a test for this?

Reply:
Done!